pkg/kubelet/eviction/helpers.go

```diff
@@ -768,7 +768,7 @@ func getResourceAllocatable(capacity v1.ResourceList, reservation v1.ResourceLis
 		}
 		return capacity.Copy(), allocate, true
 	}
-	glog.Errorf("Could not find capacity information for resource %v", resourceName)
+	glog.V(4).Infof("Could not find capacity information for resource %v, node capacity may not be initialized yet.", resourceName)
```

Comment:
Can we make this a `Warningf`?  I think the most important part is just adding that this can happen if node capacity isn't initialized.  Users shouldnt be spammed with this unless we have a bug, in which case we should fix it